replayer/connemara_replay.pl

```diff
@@ -859,23 +859,29 @@ sub switch_replay_mode_main
 {
     # build sthing like
     # database not in ('toto','tutu') or (database,source_slotname) not in ( ('tata','foo'), ('tete', 'bar'));
-    $db_list_where_clause="AND ( database not in ($db_list) or (database,source_slotname) in ($in_list) )";
+    $db_list_where_clause="WHERE ( database not in ($db_list) or (database,source_slotname) in ($in_list) )";
 }
 
 while (1)
 {
     # Let's fetch a new batch
     my $updated = 0;
     my $updated_by_pk = 0;
-    my $last_replayed;
+    # This uses the insert timestamp, for display purpose
+    my $last_replayed; # Timestamp of last replayed record (just for display)
+    # Those use the xid timestamp to determine when to commit
+    my $last_committed_epoch; # Timestamp of last commited record in seconds
```

Comment:
I would prefer seeing this initialized to 0 rather than undef and later comparing an integer to undef